plugins/autohauler.cpp

```diff
@@ -565,10 +565,20 @@ static void cleanup_state()
     labor_infos.clear();
 }
 
+static void enable_alchemist(color_ostream &out)
+{
+    if (!Units::setLaborValidity(unit_labor::ALCHEMIST, true))
+    {
+        // informational only; this is a non-fatal error
+        out.printerr("manipulator: Could not flag Alchemist as a valid skill; Alchemist will not"
```

Comment:
I think `manipulator` should probably be changed to `autohauler` - can do on my end

Reply:
whoops! thanks for the fix